trview.app/Elements/ITrigger.cpp

```diff
@@ -4,61 +4,61 @@ namespace trview
 {
     namespace
     {
-        const std::unordered_map<TriggerType, std::wstring> trigger_type_names
+        const std::unordered_map<TriggerType, std::string> trigger_type_names
         {
-            { TriggerType::Trigger, L"Trigger" },
-            { TriggerType::Pad, L"Pad" },
-            { TriggerType::Switch, L"Switch" },
-            { TriggerType::Key, L"Key" },
-            { TriggerType::Pickup, L"Pickup" },
-            { TriggerType::HeavyTrigger, L"Heavy Trigger" },
-            { TriggerType::Antipad, L"Antipad" },
-            { TriggerType::Combat, L"Combat" },
-            { TriggerType::Dummy, L"Dummy" },
-            { TriggerType::AntiTrigger, L"Antitrigger" },
-            { TriggerType::HeavySwitch, L"Heavy Switch" },
-            { TriggerType::HeavyAntiTrigger, L"Heavy Antitrigger" },
-            { TriggerType::Monkey, L"Monkey" },
-            { TriggerType::Skeleton, L"Skeleton" },
-            { TriggerType::Tightrope, L"Tightrope" },
-            { TriggerType::Crawl, L"Crawl" },
-            { TriggerType::Climb, L"Climb"}
+            { TriggerType::Trigger, "Trigger" },
+            { TriggerType::Pad, "Pad" },
+            { TriggerType::Switch, "Switch" },
+            { TriggerType::Key, "Key" },
+            { TriggerType::Pickup, "Pickup" },
+            { TriggerType::HeavyTrigger, "Heavy Trigger" },
+            { TriggerType::Antipad, "Antipad" },
+            { TriggerType::Combat, "Combat" },
+            { TriggerType::Dummy, "Dummy" },
+            { TriggerType::AntiTrigger, "Antitrigger" },
+            { TriggerType::HeavySwitch, "Heavy Switch" },
+            { TriggerType::HeavyAntiTrigger, "Heavy Antitrigger" },
+            { TriggerType::Monkey, "Monkey" },
+            { TriggerType::Skeleton, "Skeleton" },
+            { TriggerType::Tightrope, "Tightrope" },
+            { TriggerType::Crawl, "Craw" },
```

Comment:
Missing an L

Reply:
Line 24, Github decided to highlight the other lines too.

Reply:
Readded the l